RZDataBinding/NSObject+RZDataBinding.m

```diff
@@ -262,16 +263,16 @@ - (void)rz_observeBoundKeyChange:(NSDictionary *)change
     if ( boundKey != nil ) {
         id value = change[kRZDBChangeKeyNew];
 
-        [self rz_setBoundKey:boundKey withValue:value function:change[kRZDBChangeKeyBindingFunctionKey]];
+        [self rz_setBoundKey:boundKey withValue:value transform:change[kRZDBChangeKeyBindingTransformKey]];
     }
 }
 
-- (void)rz_setBoundKey:(NSString *)key withValue:(id)value function:(RZDBKeyBindingFunction)function
+- (void)rz_setBoundKey:(NSString *)key withValue:(id)value transform:(RZDBKeyBindingTransform)transform
```

Comment:
Yea, with or not with is no big deal, `bindingTransform` vs `transform` makes me think they're different.